packages/core/src/internal/indicator-container/indicator-container.scss

```diff
@@ -150,8 +150,8 @@
 
   // container children
   .ods-input-indicator ~ * {
+    cursor: pointer;
```

Comment:
This now breaks "disabled" state of both Checbox and Radio when label is set.

You would need some sort of "hack" like this:

```scss
.ods-input-indicator ~ * {
  pointer-events: none;

  a {
    pointer-events: auto;
  }
}
```

Which I personally don't like...

Reply:
Right. Other alternatives are `z-index`, or

```scss
.ods-input-indicator ~ * {
  cursor: pointer;
}

.ods-checkbox:disabled,
.ods-radio:disabled {
    + .ods-input-indicator ~ * {
  cursor: not-allowed;
}
```

Reply:
Why do you think the `a` rule is a hack though?

Reply:
Because I think it should be added to the `<a>` itself, as I mentioned - I think it's an edge case, and links by default should not be placed in labels.

Probably that your example could also work.

But please don't put `z-index` 😅

Reply:
I don't have any favorites here, they all seem pretty meh.
Probably because links in checkbox labels are real meh.

The last option is as you said, require the `<a>` inside the label to allow pointer-events to itself again.
Would you prefer that? In the story itself?

Reply:
I'm not 100% sure also.

Maybe we could do that at least for now, and then come back to this if it does not work for users.

Reply:
Sounds good. Updated with that.